sysdeps/managarm/generic/file.cpp

```diff
@@ -1506,6 +1518,12 @@ int sys_epoll_wait(int epfd, struct epoll_event *evnts, int n, int timeout, int
 	req.set_fd(epfd);
 	req.set_size(n);
 	req.set_timeout(timeout > 0 ? int64_t{timeout} * 1000000 : timeout);
+	if(sigmask != NULL) {
+		req.set_sigmask((long int)sigmask);
```

Comment:
Shouldn't we dereference `sigmask` here and use the value it points at instead of the pointer itself?